protocol/messenger_communities.go

```diff
@@ -137,6 +137,18 @@ func (m *Messenger) JoinCommunity(ctx context.Context, communityID types.HexByte
 		return nil, err
 	}
 
+	communitySettings := communities.CommunitySettings{
+		CommunityID:                  communityID.String(),
+		HistoryArchiveSupportEnabled: true,
```

Comment:
This should be `false` for now. We can make this `true` by default once we know the entire history story is robust